recipes/xtr/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+
+
+class XtrConan(ConanFile):
+    name = "xtr"
```

Comment:
I found other 2 projects using the very same name:

https://github.com/hypersoft/xtr
https://github.com/KacperKuryllo/xtr

I didn't check Bitbucket and Gitlab, but I'm fine accepting your package name as xtr only, because it's active (others seems be dead).

Reply:
Thanks

Reply:
https://repology.org/projects/x/?search=xtr 🤔 kinda maybe used? but no longer active?
Only because logging library are so common I'd prefer to see the author name

Reply:
Do you mean adding `author = "..."` ?

Reply:
Sorry, I mean the project's author as package name prefix: `name = "choll-xtr"`